src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/DRConfig/GetAzureEventHubGeoDRConfiguration.cs

```diff
@@ -0,0 +1,57 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.EventHub.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+namespace Microsoft.Azure.Commands.EventHub.Commands.GeoDR
+{
+    /// <summary>
+    /// 'Get-AzureEventHubDRConfiguration' CmdletRetrieves Alias(Disaster Recovery configuration) for primary or secondary namespace    
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, EventHubDRConfigurationVerb), OutputType(typeof(List<PSEventHubDRConfigurationAttributes>))]
+    public class GetEventHubGeoDRConfiguration : AzureEventHubsCmdletBase
+    {
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, Position = 0, HelpMessage = "Resource Group Name")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+         public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, Position = 1, HelpMessage = "Namespace Name")]
+        [ValidateNotNullOrEmpty]
+        public string Namespace { get; set; }
```

Comment:
Add piping: Get-AzureRmEventHubNamespace | Get-AzureRmEventHubDRConfiguration or Find-AzureRmResource (with Namespace) | Get-AzureRmEventHubDRConfiguration
https://github.com/Azure/azure-powershell/wiki/Piping-in-PowerShell#using-the-inputobject-parameter
Once you do this you can remove the ValueFromPipelineByPropertyName from -Namespace.

Reply:
Missing InputObject and ResourceId here.